src/test/java/com/wikia/webdriver/testcases/adstests/TestAdsTrackingPixels.java

```diff
@@ -89,6 +96,23 @@ public void adsTrackingPixelSentCuratedMainPages(String wiki, String page, Strin
     assertTrackingPixelsSent(adsBaseObject, pixelUrls);
   }
 
+  @NetworkTrafficDump
+  @Test(
+          groups = "AdsTrackingPixelsAuthPage",
+          dataProviderClass = AdsDataProvider.class,
+          dataProvider = "adsTrackingPixelsSentAuthPage"
+  )
+  @InBrowser(
+          browser = Browser.CHROME,
+          emulator = Emulator.GOOGLE_NEXUS_5
+  )
+  public void adsTrackingPixelSentAuthPage(String wiki, String page, String[] pixelUrls) {
```

Comment:
The test will be merged as part of the ticket: https://wikia-inc.atlassian.net/browse/SOC-2569